Tests/FaunaDBTests/ClientTests.swift

```diff
@@ -687,6 +687,26 @@ class ClientTests: XCTestCase {
         assert(query: BytesV(fromArray: [1, 2, 3, 4]), toReturn: BytesV(fromArray: [1, 2, 3, 4]))
     }
 
+    func testCreateFunction() {
+        let body = Query({ (a, b) in Add(a, b) })
```

Comment:
You can remove parenthesis when the lambda is the last argument of a method / function.